### PR TITLE
Feature/sar 361 d3 chart page styling

### PR DESCRIPTION
**Pull request by timAmida**
# Related Tickets

- [SAR-361](https://jira.amida-tech.com/browse/SAR-361)

# Other Repos' PR(s) Intended to Work With This PR

None

# How Things Worked (or Didn't) Before This PR
- Chart displayed ticks for every Number and Date.
- Chart displayed lines for every Number and Date.
- Date on x-axis was attached to chart.
- No labels on X and Y axis.
- Measure by line chart x axis was by measure star rating.
- X-axis values did not contain Percent.

# How Things Work Now (And How to Test)
- More Spacing on Chart Axis'
- ticks and lines only on y-axis at "0, 20, 40, 60, 80, 100", all other lines and ticks removed using DOM manipulation.
- Measure by line Styled the same as New D3-chart.
- Measure by line chart x axis was by rating out of 100.
- X-axis values contain Percents.
# Readiness

1. [x] This PR passes all automated tests.
2. [x] This PR has no linting errors.
3. [x] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.

### CODE REVIEW

**Review by eawest2 — APPROVED** [review]
Works as advertised.

**Review by anthem123 — APPROVED** [review]
Looks good and works.

Approved, but I recommend we have _d3-indicator-by-line.scss extend _d3-chart.scss sometime in the future since they are so similar.

**Comment by jmfadeley** [comment]
> Looks good and works.
> 
> Approved, but I recommend we have _d3-indicator-by-line.scss extend _d3-chart.scss sometime in the future since they are so similar.

Actually since we've got a little time, @timAmida can we do that now? Look at `_trend-display.scss` to see how it can be done.

File: src/components/Utilites/TickChange.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      ticksFound[i].innerHTML = '40%';
+    } else if (tickInnerHtml === '60') {
+      ticksFound[i].innerHTML = '60%';
+    } else if (tickInnerHtml === '80') {
+      ticksFound[i].innerHTML = '80%';
+    } else if (tickInnerHtml === '100') {
+      ticksFound[i].innerHTML = '100%';
+    } else if (tickInnerHtml === '0') {
+      ticksFound[i].innerHTML = '0%';
+    }
```
Comment (jmfadeley) [inline src/components/Utilites/TickChange.js:47]: One way that might work is to do the above is to take a modulus division of 20 to see if there's a remainder. If yes, set `ticksFound[i].style.display = 'none';` else simply append the percentage sign of `tickInnerHtml`.

```
if (tickInnerHtml % 20 > 0) {
ticksFound[i].style.display = 'none';
} else {
ticksFound[i].innerHTML = `${tickInnerHtml}%`;
}
```

File: src/components/Utilites/TickChange.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    } else if (tickLinesFound[i].__data__ === 30) {
+      tickLinesFound[i].remove();
+    } else if (tickLinesFound[i].__data__ === 50) {
+      tickLinesFound[i].remove();
+    } else if (tickLinesFound[i].__data__ === 70) {
+      tickLinesFound[i].remove();
+    } else if (tickLinesFound[i].__data__ === 90) {
+      tickLinesFound[i].remove();
+    }
+  }
```
Comment (jmfadeley) [inline src/components/Utilites/TickChange.js:77]: Same deal as above.

File: src/components/Utilites/TickChange.js (hunk 1 above)
Comment (timAmida) [reply src/components/Utilites/TickChange.js:47]: Perfect, let me put implement these changes and push.

**Pull request closed without merging**